apps/voucher/pages/api/lnurlw/[uniqueHash]/index.ts

```diff
@@ -0,0 +1,50 @@
+import { getWithdrawLinkByUniqueHashQuery } from "@/services/db"
+import { convertCentsToSats } from "@/utils/helpers"
+import { env } from "@/env"
+import { getRealtimePriceQuery } from "@/services/galoy/query/get-real-time-price"
+import { escrowApolloClient } from "@/services/galoy/client/escrow"
+const { NEXT_PUBLIC_LOCAL_URL } = env
+// TODO: Add interface for request and response objects
+
+export default async function handler(req: any, res: any) {
+  if (req.method === "GET") {
```

Comment:
instead of doing this, we should do:

```
if (req.method !== "GET") {
  return // return some error
}
```

like this we don't need to intent this function all the way to the bottom. we also exit early with the error which make the code easier to read

Reply:
Shifted to app router, not needed anymore.